### PR TITLE
feature_2054_helpEmailChanges

### LINKED ISSUE #2054: Update error messages to redirect users from the MET-Help desk to METplus Discussions.
I received the following error from MET:

> ERROR  : MetGrib2DataFile::data_plane() -> PDS template number (6) is not supported. Please email [met_help@ucar.edu](mailto:met_help@ucar.edu).

## Describe the Enhancement ##
We should update any error messages that mention met_help@ucar.edu and replace it with a link to the METplus GitHub Discussions page: https://github.com/dtcenter/METplus/discussions

### Time Estimate ###
*Estimate the amount of work required here.*
*Issues should represent approximately 1 to 3 days of work.*

### Sub-Issues ###
Consider breaking the enhancement down into sub-issues.
- [ ] *Add a checkbox for each sub-issue here.*

### Relevant Deadlines ###
*List relevant project deadlines here or state NONE.*

### Funding Source ###
2792541

### Assignee ###
- [x] Select **engineer(s)** or **no engineer** required
- [x] Select **scientist(s)** or **no scientist** required

### Labels ###
- [x] #2063
- [x] Select **priority**
- [x] Select **requestor(s)**

### Projects and Milestone ###
- [x] Select **Repository** and/or **Organization** level **Project(s)** or add **alert: NEED PROJECT ASSIGNMENT** label
- [x] Select **Milestone** as the next official version or **Future Versions**

## Define Related Issue(s) ##
Consider the impact to the other METplus components.
- [ ] [METplus](https://github.com/dtcenter/METplus/issues/new/choose), [MET](https://github.com/dtcenter/MET/issues/new/choose), [METdatadb](https://github.com/dtcenter/METdatadb/issues/new/choose), [METviewer](https://github.com/dtcenter/METviewer/issues/new/choose), [METexpress](https://github.com/dtcenter/METexpress/issues/new/choose), [METcalcpy](https://github.com/dtcenter/METcalcpy/issues/new/choose), [METplotpy](https://github.com/dtcenter/METplotpy/issues/new/choose)

## Enhancement Checklist ##
See the [METplus Workflow](https://metplus.readthedocs.io/en/latest/Contributors_Guide/github_workflow.html) for details.
- [x] Complete the issue definition above, including the **Time Estimate** and **Funding Source**.
- [x] Fork this repository or create a branch of **develop**.
Branch name: `feature_<Issue Number>_<Description>`
- [x] Complete the development and test your changes.
- [ ] Add/update log messages for easier debugging.
- [ ] Add/update unit tests.
- [ ] Add/update documentation.
- [x] Push local changes to GitHub.
- [x] Submit a pull request to merge into **develop**.
Pull request: `feature <Issue Number> <Description>`
- [x] Define the pull request metadata, as permissions allow.
Select: **Reviewer(s)** and **Linked issues**
Select: **Repository** level development cycle **Project** for the next official release
Select: **Milestone** as the next official version
- [x] Iterate until the reviewer(s) accept and merge your changes.
- [x] Delete your fork or branch.
- [x] Close this issue.

### PR DESCRIPTION
**Pull request by davidalbo**
## Expected Differences ##

- [x] Do these changes introduce new tools, command line arguments, or configuration file options? **[No]**</br>
If **yes**, please describe:</br>

- [x] Do these changes modify the structure of existing or add new output data types (e.g. statistic line types or NetCDF variables)? **[No]*</br>
If **yes**, please describe:</br>

## Pull Request Testing ##

- [x] Describe testing already performed for these changes:  None so far</br>

- [x] Recommend testing for the reviewer(s) to perform, including the location of input datasets, and any additional instructions: Some kind of standard tests</br>

- [x] Do these changes include sufficient documentation updates, ensuring that no errors or warnings exist in the build of the documentation? **[No]**

- [x] Do these changes include sufficient testing updates? **[No]**

- [x] Will this PR result in changes to the test suite? **[No]**</br>
If **yes**, describe the new output and/or changes to the existing output:</br>

- [x] Please complete this pull request review by **[2022/02/25]**.</br>

## Pull Request Checklist ##
See the [METplus Workflow](https://metplus.readthedocs.io/en/latest/Contributors_Guide/github_workflow.html) for details.
- [x] Complete the PR definition above.
- [x] Ensure the PR title matches the feature or bugfix branch name.
- [x] Define the PR metadata, as permissions allow.
Select: **Reviewer(s)**
Select: **Organization** level software support **Project** or **Repository** level development cycle **Project**
Select: **Milestone** as the version that will include these changes
- [x] After submitting the PR, select **Linked issues** with the original issue number.
- [ ] After the PR is approved, merge your changes. If permissions do not allow this, request that the reviewer do the merge.
- [ ] Close the linked issue and delete your feature or bugfix branch from GitHub.

### CODE REVIEW

File: met/README [hunk 1]
```diff
@@ -22,7 +22,7 @@ This is the main directory for the Model Evaluation Tools source code release.
 For questions, please:
 - Refer to the MET User's Guide: https://met.readthedocs.io/en/latest
 - Refer to the MET website: http://dtcenter.org/community-code/model-evaluation-tools-met
-- Send mail to met_help@ucar.edu.
+- Interact with the METplus GitHub Discussions page: https://github.com/dtcenter/METplus/discussions
```
Comment (jprestop) [inline met/README]:
```suggestion
- Create a post in the METplus GitHub Discussions forum: https://github.com/dtcenter/METplus/discussions
```

File: met/docs/Users_Guide/appendixA.rst [hunk 2]
```diff
@@ -1777,9 +1777,8 @@ Try the following 2 things:
 		
 		  MET_BUFRLIB=/home/username/BUFRLIB_v10.2.3 
 
-After doing that, please try recompiling MET. If it fails,
-please send met_help@ucar.edu the following log files.
-"make_install.log" as well as "config.log".
+After doing that, please try recompiling MET. If it fails, please submit the following log files: "make_install.log" as well as "config.log".them to `METplus GitHub Discussions Forum <https://github.com/dtcenter/METplus/discussions>`_
```
Comment (jprestop) [inline met/docs/Users_Guide/appendixA.rst]:
```suggestion
After doing that, please try recompiling MET. If it fails, please submit the "make_install.log" file as well as the "config.log" file with a new post in the `METplus GitHub Discussions Forum <https://github.com/dtcenter/METplus/discussions>`_.
```

File: met/src/libcode/vx_data2d_grib2/data2d_grib2.cc [hunk 3]
```diff
@@ -704,7 +704,7 @@ void MetGrib2DataFile::read_grib2_record_list() {
             mlog << Error << "\nMetGrib2DataFile::data_plane() -> "
                  << "PDS template number ("
                  << gfld->ipdtnum << ") is not supported. "
-                 << "Please email met_help@ucar.edu.\n\n";
+                 << "Please take this to the METplus GitHub Discussions page: https://github.com/dtcenter/METplus/discussions\n\n";
```
Comment (jprestop) [inline met/src/libcode/vx_data2d_grib2/data2d_grib2.cc]:
```suggestion
                 << "Please create a new post with this information in the METplus GitHub Discussions forum at https://github.com/dtcenter/METplus/discussions\n\n";
```

**Review by jprestop — COMMENTED** [review]
@davidalbo Hi Dave.  I made some suggested changes.  You should be able to just click the "Commit suggestion" button if you agree with the suggested wording and then resolve the conversation by clicking "Resolve Conversation".  See screenshot.
Screen Shot 2022-02-22 at 3 16 29 PM

**Review by jprestop — CHANGES_REQUESTED** [review]
@davidalbo When you're finished with any suggested changes, you can click the circular arrow icon after my name to let me know you're ready for me to re-review.  See screenshot.  Please let me know if you have any questions.
Screen Shot 2022-02-22 at 3 19 24 PM

**Comment by davidalbo** [comment]
Julie, I believe I went with all your suggested changes. Let me know if not the case.   It seems a little 'messy' to me, in the sense that mistakes might be made (by me), as a rookie.

**Comment by jprestop** [comment]
@davidalbo Some other comments...

I noticed that the boxes in the PR were marked as checked (with the "x" in the brackets, e.g. [x]) under "Pull Request Checklist", but not under "Expected Difference" and "Pull Request Testing" for the PR weren't checked.  I went ahead and checked them. 

For "Describe testing already performed for these changes:", you said, "None so far".  Some things you could have done to see how it looked were to take a look at these lines on these pages to see how your change looked:
https://github.com/dtcenter/MET/blob/0f2d6febbeb1894341f157b054b5904e8255b6cc/met/README#L25
https://github.com/dtcenter/MET/blob/0f2d6febbeb1894341f157b054b5904e8255b6cc/met/src/libcode/vx_data2d_grib2/data2d_grib2.cc#L707
You could also view the changes in your branch on this page of the documentation:
https://met.readthedocs.io/en/feature_2054_helpemailchange/Users_Guide/appendixA.html

Since GitHub Actions is set up, you didn't really have to do any testing - yay!  And, my apologies for not suggesting these things earlier via email.  Without see the affected files, it was difficult to come up with these things off the top of my head.  It's much easier here with this PR right in front of me.

Under "Recommend testing for the reviewer(s) to perform, including the location of input datasets, and any additional instructions", since the standard tests are being run automatically with GitHub Actions, you could suggest that the reviewer review the Files Changed and ensure that all of the tests passed.

Instead of answering "No" to "Do these changes include sufficient documentation updates, ensuring that no errors or warnings exist in the build of the documentation?" and "Do these changes include sufficient testing updates?", I would have said, "Yes" and "N/A".  Regarding the documentation updates, you likely would have noticed if you received errors in the documentation from your push to GitHub (you would have received an email saying that Actions failed).  Regarding the testing, you didn't need to add any tests so the testing updates question really isn't applicable here.

**Review by jprestop — APPROVED** [review]
@davidalbo Thanks for making the suggested changes!  This looks great!  Thank you also for requesting the re-review.  I approve this pull request.  You should be all set to merge your changes, close the linked issue, and delete your feature branch from GitHub.  Please let me know if you have any questions and thanks for your work on this task!

**Comment by jprestop** [comment]
@davidalbo, well, that is, once all of the tests finish running...

**Pull request merged**